File: Excercises/divisible.py
number = int(input())
nlist = input().split(" ")
for a in range(len(nlist)):
  nlist[a] = int(nlist[a])

if all(number % x == 0 for x in nlist):
  print('divisible by all')
else:
  print('not divisible by all')

File: Ruby/divisible.rb
n = gets.chomp.to_i
l = gets.chomp.split()
l = l.map(&:to_i)
t = l.all? { |x| n%x == 0 }
if t == true
  puts'divisible by all'
else
  puts 'not divisible by all'
end